views/style.css

```diff
@@ -384,3 +384,29 @@ h5 {
   display: flex;
   justify-content: space-between;
 }
+
+.tooltip{
+    display:inline;
+    position:relative;
+  }
+.tooltip:hover{text-decoration:none}
+
+.tooltip:active:after{
```

Comment:
Make the background black and the text white. Have an 18-px bottom border to make it appear above instead of below the question mark. max-width:200px is my attempt to make the box wider, but it didn't really work